crawl-ref/source/dat/descript/de/gods.txt

```diff
@@ -121,16 +121,6 @@ verzehren, die von ihren Anhängern woanders im Verlies liegen gelassen wurden.
 Besonders strenggläubige Anhänger werden genauso formlos wie ihr Gott, weil
 ihre Eigenschaften und Mutationen sich unter Jiyvas Einfluss verändern.
 %%%%
-Zin powers
```

Comment:
I believe the idea is translators would look at a different of the file, see the entry is deleted and then know the entry needs an update -- it's stable version focused IOW. But I can readd if you prefer